src/questionnaire.js

```diff
@@ -9,6 +9,14 @@ module.exports = [
         return "Project name may only include letters, numbers, underscores and hashes.";
     },
   },
+  {
+    name: "projectDirectoryPath",
+    type: "input",
+    message: "Enter destination folder path?",
+    when: (answers) => {
+      return answers.projectName;
+    },
+  },
```

Comment:
do we have a fallback, if entered path is not available in system? or someone miss typed or pasted wrong path, will he get errors?